packages/mobile/src/components/core/FlatList.tsx

```diff
@@ -74,6 +90,15 @@ const AnimatedFlatList = forwardRef<RNFlatList, FlatListProps>(
         <Animated.FlatList
           {...other}
           scrollToOverflowEnabled
+          refreshControl={
+            Platform.OS === 'ios' ? undefined : (
+              <RefreshControl
+                refreshing={!!isRefreshing}
+                onRefresh={onRefresh ?? undefined}
+                colors={[neutral]}
+              />
```

Comment:
kinda a nit, but maybe we make this a base component that all the lists can use? handles color, maybe also platform etc?

Reply:
That was my first approach, but for some reason having anything other than `RefreshControl` at the top level of this value breaks the entire flatlist. Might have to do with how it's interfacing with native components. This code is necessarily WET

Reply:
oh woww interesting! well thanks for the context